### DISCUSSION TITLE
Grupo hackatón

### DISCUSSION THREAD

**Discussion by oayala-1** [Launch-X-Latam/MisionFrontEnd#553] [Team Posts]
Hola a todos, busco un grupo para el hackaton, mi correo es: lx05422@innovaccion.mx

**Comment by AlvaroFierro** [comment]
hola, si no tienes equipo podemos armar uno, lx03108@innovaccion.mx

**Comment by CrisQF** [comment]
Me puedo unir a su equipo? Mi correo es lx05668@innovaccion.mx

**Comment by AndresLunaH** [comment]
Hola, podría unirme a su equipo? Mi correo es: lx15493@innovaccion.mx

**Comment by MercedesMV** [comment]
Hola! Soy Mercedes yo no tengo equipo, aún hay espacio disponible?. Este es mi correo: lx16992@innovaccion.mx

**Comment by oayala-1** [comment]
Claro, en un momento les mando un correo :)

**Comment by Javilh97** [comment]
ya estan completos? Tambien estoy buscando equipo :p

**Comment by oayala-1** [comment]
> ya estan completos? Tambien estoy buscando equipo :p

Dos de los que aquí comentaron no respondieron, pudiera añadirte, deja aquí tu correo innovaccion para contactarme contigo :)

**Comment by oayala-1** [comment]
> Si aún hay lugar quisiera unirme.
Dos de los que aquí comentaron no respondieron, pudiera añadirte, deja aquí tu correo innovaccion para contactarme contigo :)

**Comment by TonatiuhDSJ** [comment]
todavía tienen cupo?? y si es así me puedo unir?? mi correo es: lx15933@innovaccion.mx.

**Comment by oayala-1** [comment]
> todavía tienen cupo?? y si es así me puedo unir?? mi correo es: lx15933@innovaccion.mx.

Claro, en un momento te mando un correo :)

**Comment by TonatiuhDSJ** [comment]
@oayala-1 Muy bien, muchas gracias 😄

**Comment by Illescas970103** [comment]
Misma pregunta?, tambien busco equipo para hackaton mi correo es Ix00645@innovaccion.mx
